### PR TITLE
[doc] Add "Dependency" anchor link to quick start section

### PR DESCRIPTION
**Pull request by nchelluri**
The quick start section is already called "For the Impatient", and this
minor doc tweak should decrease the time and effort demanded of a
new user before their first run by a very tiny amount, and it makes the
README a touch more cohesive.

[ci skip]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ and easy to use.
 
 ## For the Impatient
 
-I know, you just want to build and play.  If you have all the dependencies
-installed (see below), then simply do this:
+I know, you just want to build and play.  If you have all the [dependencies
+installed](#dependencies) (see below), then simply do this:
```
Comment (afh) [inline README.md]: In my opinion `installed` should not be part of the link.
If `dependencies` is a link then I find `(see below)` obsolete.
Another possibility would be to make `see below` a link.

Just my 2¢

Comment (nchelluri) [inline README.md]: Have updated the PR. Thoughts?

**Comment by afh** [comment]
Please squash the three commits into one keeping the `[ci skip]` line.

**Comment by nchelluri** [comment]
Should be good now. I wish I could rename the branch but I think that'll close the PR and I'll have to create a new one. Happy to do that if you'd prefer. Thinking of a name like `minor-doc-tweak`.

**Comment by afh** [comment]
It's fine like this and I merged it. Thank you for your contribution to the project! :+1:

**Pull request merged**